### DISCUSSION TITLE
Participate in the MNF mobile ecosystem survey!

### DISCUSSION THREAD

**Discussion by keith** [MobileNativeFoundation/discussions#185] [General]
tl;dr: please take the MNF's quick mobile ecosystem survey https://www.surveymonkey.com/r/87QMTDL, results and raw data will be shared once they're available.

Hey everyone, last week the Mobile Native Foundation launched a [public survey](https://www.surveymonkey.com/r/87QMTDL) to try and get a pulse on different trends throughout the mobile community. Specifically we're very interested in seeing how trends change based on things like company size, number of apps, and other things like that. We're hoping that these trends shed some light on areas where the community is moving, and where there could be more investment in open source projects. We plan to analyze the data from the survey and share the results here, as well as sharing the raw data for folks who want to do extra analysis.

**We can't do this without you!** So please spend a few minutes to fill out the survey https://www.surveymonkey.com/r/87QMTDL

Thanks!

**Comment by keith** [comment]
Here's a quick update with the results https://github.com/MobileNativeFoundation/discussions/discussions/191
